### PR TITLE
Add testing on django 1.8

### PR DESCRIPTION
**Pull request by nikolas**

### CODE REVIEW

**Comment by troygrosfield** [comment]
:+1:

File: tox.ini [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
     -r{toxinidir}/requirements/default.txt
     -r{toxinidir}/requirements/test.txt
-    1.4: Django==1.4.18
+    1.4: Django==1.4.20
     1.5: Django==1.5.12
-    1.6: Django==1.6.10
-    1.7: Django==1.7.3
+    1.6: Django==1.6.11
+    1.7: Django==1.7.7
+    1.8: Django==1.8
```
Comment (troygrosfield) [inline tox.ini]: django is now at 1.8.2 (I know this was an older PR).

**Comment by troygrosfield** [comment]
The pull seems straightforward enough.  

@ask thoughts on merging?

**Comment by troygrosfield** [comment]
django-celery has been in alpha mode since June 2014 and django-celery hasn't seen any release since 9/9/2014.  What's the plan for releasing 3.2?

**Comment by thedrow** [comment]
@troygrosfield We don't have enough features ready yet for 3.2 and a lot of bug fixes to do before even implementing them.
Feel free to help.

**Comment by troygrosfield** [comment]
Fair enough. I actually care less about a "3.2" release than I do about this repo just being django 1.8 ready since 1.8 has been out for sometime now.  This was at least a step in the right direction for giving future changes the needed transparency knowing they won't break anything in django 1.8.

With that said,  is there a specific list of critical issues somewhere you guys are looking for help on? 

Thanks in advance for all your guys work on this repo! We have been following it for quite a long time and appreciate all the time and effort you guys (contributors) put in!

**Comment by thedrow** [comment]
@troygrosfield These are urgent bugs we need to identify what exactly is failing and why and provide fixes for:
https://github.com/celery/celery/labels/Worker%20Hangs
https://github.com/celery/celery/labels/Autoscaling
https://github.com/celery/celery/labels/Canvas

**Pull request merged**